src/Pages/juheejang/Login/Login.js

```diff
@@ -1,11 +1,108 @@
-import React from "react";
+import React from 'react';
 import "./Login.scss";
-import { withRouter } from "react-router-dom";
+import { Link } from 'react-router-dom';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+      loginBtnOpacity: false,
+      loginBtn: true
+    };
+  }
+
+  handleIdInput = (event) => {
+    this.setState({
+      idValue: event.target.value,
+    })
+
+    if(event.target.value.includes("@") && this.state.pwValue.length >= 5){
+      this.setState({
+        loginBtnOpacity: true,
+        loginBtn: false
+      })
+    } else {
+      this.setState({
+        loginBtnOpacity: false,
+        loginBtn: true
+      })
+    }
+    console.log("입력된 id값 : "+event.target.value);
+  }
+
+  handlePwInput = (event) => {
+    this.setState({
+      pwValue: event.target.value,
+    })
```

Comment:
비구조화 알려드리겠습니다! 좀 더 간결하게 사용하기 위해섭니다!

```suggestion
    const { value } = event.target;
    this.setState({
      pwValue: value,
    })
```

Reply:
확인 후 수정했습니다 😀